src/js/experiments/gettingStartedGuide.js

```diff
@@ -23,30 +23,48 @@ function showHomePageBadges() {
   }
 }
 
-function setUpTracking() {
-  const badges = Array.from($('.wrapper-link'));
-  badges.forEach((badge) => {
-    badge.addEventListener('click', () => {
-      window.AnalyticsClient.trackAction('clicked-landing-page-badge', {
-        badgeText: badge.innerText,
-        badgeHref: badge.href,
+function setUpTracking(variation) {
+  // Adding tracking to a tags that link to /docs/2.0/getting-started/
+  const newBadges = Array.from($("li > a[href='/docs/2.0/getting-started/']"));
+  newBadges?.forEach((link, i) => {
+    let location;
+    if (i === 0) location = 'mobile-sidebar';
```

Comment:
How do we know 0 is mobile? 1 is sidebar 2 is homepage? ect? Is this deterministic?

Reply:
I think its based on how the order of the HTML elements in the DOM and how they are parsed by the browser.